src/blog/2023/011-form-js-1.6.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Unveiling Dynamic Forms
+description: "Explore what's new in form-js@1.6.0: Unveil the power of dynamic lists and an innovative table component, designed to enhance the functionality and versatility of your forms."
+preview_image: http://bpmn.io/assets/attachments/blog/2023/011-dynamic-table.png
+layout: blogpost.hbs
+slug: 2023-form-js-1-6-0-unveiling-dynamic-forms
+author:
+  - Vinicius Goulart <https://github.com/vsgoulart>
+  - Valentin Serra <https://github.com/skaiir>
+published: 2023-12-18 12:00
+
+releases:
+  - "form-js@1.6.0"
+---
+
+<p class="introduction">
+  We're thrilled to announce the [form-js@1.6.0 release](https://github.com/bpmn-io/form-js), a significant leap in form development. This version introduces dynamic lists and a new table component, elevating the capabilities of complex form creation.
+</p>
+
+<!-- continue -->
+
+Explore these new features [on our demo](https://demo.bpmn.io/form) or using the [Camunda form playground](https://github.com/camunda/form-playground).
+
+## Dynamic Lists
+
+Without manual work, dynamic lists enable the effortless generation of repetitive sets of form components, such as data tables or ordered lists.
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/011-dynamic-list.png">
+
+  <p class="caption">
+    Dynamic lists offer flexibility in modeling repetitive lists of form components.
+  </p>
+</div>
+
+## Table Component
+
+The new table component in `form-js@1.6.0` brings an improved approach to data organization within forms, offering a structured and user-friendly interface. This addition simplifies the management and entry of complex data, simplifying overall form usability.
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/011-table-component.png">
+
+  <p class="caption">
+    Organize tabular data with the new table component, giving your users all the information they need to fill their forms in a space efficient way.
+  </p>
+</div>
+
```

Comment:
Feature how the data source can be static (just input data) or computed from existing form data, i.e. a prior dynamic list.

Reply:
Consider featuring how (also more complex) nested components can be stacked inside a dynamic list.